lib/chef/knife/kvm_vm_create.rb

```diff
@@ -245,14 +250,16 @@ def create_vm
         result = run_remote_command(command, true)
         ui.info result
 
-        # bootstrap things now
-        if config[:bootstrap_node_ip]
-          bootstrap_node(config[:bootstrap_node_ip])
-        else
-          bootstrap_node(config[:guest_ip])
+        unless config[:skip_bootstrap]
```

Comment:
should this be == false ?

Reply:
Nope - skip_bootstrap will be true if the option is specified, so it will skip it here.